anomaly_detector/exception/exceptions.py

```diff
@@ -7,3 +7,19 @@ class factStoreEnvVarNotSetException(Exception):
     def __init__(self, msg="fact store url env var not set"):
         """Initialize message."""
         self.message = msg
+
+
+class ModelLoadException(Exception):
+    """Validates that model has been loaded."""
+
+    def __intit__(self, msg="There is no existing model to load"):
+        """Initialize message."""
+        self.message = msg
+
+
+class ModelSaveException(Exception):
+    """Validates that model has been saved."""
+
+    def __intit__(self, msg="The model could not be saved"):
+        """Initialize message."""
+        self.message = msg
```

Comment:
This is a good simple message but what will be the case when we are using multiple models. For now I'm merging this but we may have to revisit when we have more models. Different models being trained.